torch/_export/graph_module.py

```diff
@@ -0,0 +1,149 @@
+# pyre-strict
```

Comment:
Maybe we can get rid of all the pyre stuff?

Reply:
Good call, done.